src/app/lib/utilities/getUrlHelpers/getEmbedUrl/index.test.js

```diff
@@ -283,3 +283,87 @@ describe('makeAbsolute makes a relative URL absolute', () => {
     delete process.env.SIMORGH_APP_ENV;
   });
 });
+
+describe('buildAvEmbedURL', () => {
+  describe('service with variant', () => {
+    it('should build an embed URL for the page', () => {
+      expect(
+        getEmbedUrl({
+          mediaId: '/serbian/cyr/av-embeds/srbija-68707945',
+          type: 'avEmbed',
+          queryString: '',
+        }),
+      ).toEqual('https://www.bbc.com/serbian/cyr/av-embeds/srbija-68707945');
+    });
+
+    it('should build an embed URL for the page omitting renderer env', () => {
+      expect(
+        getEmbedUrl({
+          mediaId: '/russian/av-embeds/features-49881797?renderer_env=live',
```

Comment:
Technically this should not belong in the "service with variant" section - can we move it?